sinks/src/main/scala/com.snowplowanalytics.snowplow.eventgen/Config.scala

```diff
@@ -126,6 +127,9 @@ object Config {
   implicit val outputDecoder: Decoder[Output] =
     deriveConfiguredDecoder[Output]
 
+  implicit val contextConfigDecoder: Decoder[Context.ContextsConfig] =
```

Comment:
What happens if the config has min > max?

Don't think we necessarily need to worry about it, but if it's easy to enforce and throw on boot up if this is wrong that might be nice.

Absolutely wouldn't suggest it's necessary if it's non-trivial to do so. If it's non trivial I wouldn't suggest it's necessary to do it at all - this PR already meets the main purpose of facilitating internal testing, the rest is just a nice to have. (I've personally implemented much less safe config into this project so am aware of the hypocrisy of the suggestion :D )

Reply:
Added a validation on the config decoder.